pkg/kapp/resources/resources.go

```diff
@@ -401,6 +401,18 @@ func (c *ResourcesImpl) Exists(resource Resource) (bool, error) {
 			return isDone, c.resourceErr(err, "Checking existence of", resource)
 		}
 
+		if existsOpts.checkForSameUID() {
```

Comment:
instead of having a method that returns the field value, it's probably better to just use existsOps.SameUID directly.

Reply:
I was thinking adding a function with name existsOpts.checkForSameUID() gives more context on what we want to do as compared to existsOps.SameUID. Your thoughts ?